### DISCUSSION TITLE
After transcription, there is still occupation of GPU memory

### DISCUSSION THREAD

**Discussion by lesrose** [SYSTRAN/faster-whisper#886] [Q&A]
`from faster_whisper import WhisperModel

model = WhisperModel("/data/ai-model/faster-whisper-large-v3")
segments, info = model.transcribe("/data/resource-data/2024-06-19/gpu1/b5fa231dfc7a462d8c91dbbc36cd7c72.mp3")
for i in range(10):
    time.sleep(10)`
    Why is there still gpu memory usage during sleep
    
![image](https://github.com/SYSTRAN/faster-whisper/assets/48045297/c27f5118-c44f-4428-8dca-3c417c98f6b9)

**Comment by toanhuynhnguyen** [comment]
You should implement this before the time.sleep(10):
```bash
del model
gc.collect()
for i in range(10):
    time.sleep(10)
```

gc is imported as below:
```bash
import gc
```
